otx/algorithms/detection/task.py

```diff
@@ -472,44 +500,61 @@ def _det_add_predictions_to_dataset(self, all_results, width, height, confidence
                     continue
 
                 assigned_label = [ScoredLabel(self._labels[label_idx], probability=probability)]
-                if coords[3] - coords[1] <= 0 or coords[2] - coords[0] <= 0:
-                    continue
```

Comment:
I think this statement should be applied to ellipse too

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2172/commits/3cdd32725a5155c9e5f304c6cb372a87bafee917